module/apmgopg/hook.go

```diff
@@ -25,26 +27,18 @@ import (
 
 const elasticApmSpanKey = "go-apm-agent:span"
 
+// QueryHook is an implementation of pg.QueryHook that reports queries as spans to Elastic APM.
 type QueryHook struct {
 	Database string
 	Username string
-
-	UseFormattedQuery bool
 }
 
+// BeforeQuery initiates the span for the database query
 func (qh *QueryHook) BeforeQuery(evt *pg.QueryEvent) {
-	var sql string
-	var err error
-
-	if qh.UseFormattedQuery {
-		sql, err = evt.FormattedQuery()
-	} else {
-		sql, err = evt.UnformattedQuery()
-	}
-
+	sql, err := evt.UnformattedQuery()
 	if err != nil {
-		// todo: panic/logging ?
-		return
+		// Expose the error making it a bit easier to debug
+		sql = fmt.Sprintf("[go-pg] error: %s", err.Error())
 	}
 
 	span, _ := apm.StartSpan(evt.DB.Context(), apmsql.QuerySignarture(sql), "db.postgres.query")
```

Comment:
can you please make it "db.postgresql.query", to match the name we use in apmsql:

https://github.com/elastic/apm-agent-go/blob/e4cbc85a0ea49329a88055d657e5a3892761e683/internal/sqlutil/drivername.go#L42